### PR TITLE
Support for GitHub API tokens added (fixes Hexxeh/rpi-update#151)

### PR DESCRIPTION
**Pull request by darkwinternight**
Use it like this:

$ sudo GITHUB_API_TOKEN=“<my-api-token”> rpi-update

### CODE REVIEW

**Comment by popcornmix** [comment]
Looks fine. Can you add a description of the setting to README.md?

**Comment by darkwinternight** [comment]
PR updated @popcornmix

File: README.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ By default, `rpi-update` will not use a custom GitHub API token. If you run into
 
 	sudo GITHUB_API_TOKEN=<your API token> rpi-update
 
+## Troubleshooting
```
Comment (darkwinternight) [inline README.md]: This used to be the same headline level as the expert options.

**Comment by popcornmix** [comment]
Thanks!

**Pull request merged**